external/src/main/java/com/redhat/cloud/policies/engine/lightweight/PolicyToTriggerConverter.java

```diff
@@ -0,0 +1,34 @@
+package com.redhat.cloud.policies.engine.lightweight;
+
+import org.hawkular.alerts.api.model.condition.Condition;
+import org.hawkular.alerts.api.model.condition.EventCondition;
+import org.hawkular.alerts.api.model.trigger.FullTrigger;
+import org.hawkular.alerts.api.model.trigger.Trigger;
+
+import java.util.List;
+
+public class PolicyToTriggerConverter {
+
+    private static final String NO_LONGER_USED = "";
+
+    public static FullTrigger convert(Policy policy) {
+        FullTrigger fullTrigger = new FullTrigger();
+        fullTrigger.setTrigger(buildTrigger(policy));
+        fullTrigger.setConditions(List.of(buildCondition(fullTrigger.getTrigger(), policy.condition)));
+        return fullTrigger;
```

Comment:
I just noticed that this method is not loading the `policy.actions`.
So right now we could have:
- no actions
- notification action

In the future we are expecting more actions such as assigning a tag to the host. 

As i understand, right now this assumes all the policies should send a notification.

Reply:
I forgot that a trigger could have no action in the UI 🤦‍♂️  I'll fix that and put the action check you mentioned in the other comment back in the code.

Reply:
Actions should now be loaded from the DB and checked before sending a notification.